README.rst

```diff
@@ -254,6 +254,17 @@ would be default in next MAJOR release (unified differ, estimating, full-width p
 
     $ PHP_CS_FIXER_FUTURE_MODE=1 php php-cs-fixer.phar fix -v --diff
 
+Rules
+-----
+
+Use the following command to quickly understand what a rule will do to your code:
+
+.. code-block:: bash
+
+.. code-block:: bash
+
+    $ php php-cs-fixer.phar describe align_multiline_comment
```

Comment:
maybe mention the ruleset as well, eg `php-cs-fixer describe @PSR2` ?

Reply:
good idea! I added it.